### PR TITLE
ACAS-578

### PR DESCRIPTION
**Pull request by brianbolt**
To be reviewed after #117 

## Description
 - Adds get_all_meta_lots to get all meta lots and optionally filter by project code
 - Adds get_meta_lots_by_lot_corp_names to get all metalots by corp name

## Related Issue
ACAS-578

## How Has This Been Tested?
acasclient tests

### CODE REVIEW

**Review by dalejerikson — APPROVED** [review]
👍

File: acasclient/acasclient.py [hunk 1]
```diff
@@ -417,7 +421,51 @@ def get_all_lots(self):
...
+        Args:
+            lot_corp_names (list): A list of lot corp names
+
+        Returns: Returns an array of dict objects respresenting meta lots
+        """
+        meta_lots = []
+
+        # Call get_meta_lot for each lot corp name
+        # Run max_workers requests in parallel
+        with ThreadPoolExecutor(max_workers=max_workers) as executor:
```
Comment (bffrost) [inline acasclient/acasclient.py:465]: This is okay for now, but we may want to implement this on the server side in a single route if we start seeing performance issues.

**Review by bffrost — APPROVED** [review]

**Comment by bffrost** [comment]
@brianbolt the automated CI tests are failing for this PR, and it does seem related to your new test. So please take a look and fix that before merging.

File: acasclient/acasclient.py (hunk 1 above)
Comment (brianbolt) [reply acasclient/acasclient.py:465]: Yeah, I didn't want to mess with these routes or spend too much time on it.  I'll make a comment in the code about it.

**Comment by brianbolt** [comment]
Fixed the tests with a workaround.  It worked locally because I was running the test independent of all other tests and the issue only came up when running in conjunction with other bulk load files tests.  I filed ACAS-581 to address the underlying issue that I workedaround.

**Pull request merged**